Security_Engine/Create/CameraFieldOfView.cs

```diff
@@ -0,0 +1,364 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+using BH.oM.Geometry;
+using BH.oM.Security.Elements;
+using BH.oM.Base.Attributes;
+using System.Collections.Generic;
+using BH.Engine.Geometry;
+using System;
+using System.Linq;
+
+namespace BH.Engine.Security
+{
+    public static partial class Create
```

Comment:
Please distribute the methods into relevant classes and files. Personally, I would put the main method in `Compute` - `Create` usually contains lightweight, simple methods that only assign properties etc. This one is a proper heavyweight on the other hand.

Reply:
Agreed 👍